website/cue/reference/components/transforms/remap.cue

```diff
@@ -157,17 +157,27 @@ components: transforms: "remap": {
 		event_data_model: {
 			title: "Event Data Model"
 			body:  """
-				You can use the `remap` transform with both log and metric events.
+				You can use the `remap` transform to handle both log and metric events.
 
 				Log events in the `remap` transform correspond directly to Vector's [log schema](\(urls.vector_log)),
-				which means that the transform has access to the whole event.
-
-				With metric events the remap transform has:
-
-				* read-only access to the event's`.type`
-				* read/write access to `kind`, but it can only be set to one of `incremental` or `absolute` and cannot be deleted
-				* read/write access to `name`, but it cannot be deleted
-				* read/write/delete access to `namespace`, `timestamp`, and keys in `tags`
+				which means that the transform has access to the whole event and no restrictions on how the event can be
+				modified.
+
+				With [metric events](\(urls.vector_metric)), VRL is much more restrictive. Below is a field-by-field
+				breakdown of VRL's access to metrics:
+
+				Field | Access | Specific restrictions (if any)
+				:-----|:-------|:------------------------------
+				`type` | Read only |
+				`kind` | Read/write | You can set `kind` to either `incremental` or `absolute` but not to an arbitrary value.
+				`name` | Read/write |
+				`timestamp` | Delete |
```

Comment:
You should be able to read/write/delete `timestamp` ([code](https://github.com/vectordotdev/vector/blob/master/lib/vector-core/src/event/vrl_target.rs). The value you are inserting, though, has to be a VRL timestamp otherwise it is ignored.

Reply:
Updated